snapcraft/tests/plugins/test_python.py

```diff
@@ -61,6 +64,123 @@ def fake_empty_pip_list(*args, **kwargs):
         return ''
 
 
+class FakePip(fixtures.Fixture):
```

Comment:
put this in `snapcraft.tests.plugins.fakes` or something.

Reply:
I've moved it to snapcraft/tests/fixture_setup/fake_pip.py, imported as snapcraft.tests.fixture_setup.FakePip. How about that?